types/knex-cleaner/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for knex-cleaner 1.3
+// Project: https://github.com/steven-ferguson/knex-cleaner
+// Definitions by: Karol Goraus <https://github.com/Szarlus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import * as Knex from 'knex';
+
+export interface KnexCleanerOptions {
+    mode?: 'truncate' | 'delete'; // Can be ['truncate', 'delete'], defaults to 'truncate
```

Comment:
Misleading comment because it suggests that it can be an array. I would leave only `// default is 'truncate'`

Reply:
Oh yeah, my bad. I took it from the original package README and then added string literals as type without updating the comment. Thanks a lot